ui/src/components/page_navigation/PageNavigation.js

```diff
@@ -85,7 +85,6 @@ const MoreActionsButton = ({ actions }) => {
       panelPaddingSize="none"
       anchorPosition="downRight">
       <EuiContextMenuPanel
-        hasFocus={false}
```

Comment:
Removed this because `hasFocus` is no longer a prop for `EuiContextMenuPanel` (see https://elastic.github.io/eui/#/navigation/context-menu).